erts/doc/src/absform.xml

```diff
@@ -200,6 +200,9 @@
           </p>
         </item>
       </list>
+      <p>See <seetype marker="stdlib:erl_parse#form_info">
+        <c>the form_info/0</c> type</seetype> in
```

Comment:
I think  "type" is meant to be outside of the <seetype> tag. What at it: the <c> tag is missing in the <seeerl> just above this line.